src/components/picker/index.tsx

```diff
@@ -144,14 +144,13 @@ export class SmoothlyPicker implements Clearable, Editable, Input, ComponentDidL
 	areValuesEqual(selected: Map<any, Option>, initialValue: Map<any, Option>): boolean {
 		const initialValueArray = Array.from(initialValue.values(), option => option.value)
 		const selectedArray = Array.from(selected.values(), option => option.value)
+		let result: boolean
 		if (selectedArray.length !== initialValueArray.length) {
-			return false
+			result = false
+		} else {
```

Comment:
When there is only one statement inside the if/else block we can cut the braces turning 
```ts
if (selectedArray.length !== initialValueArray.length) {
    result = false
} else {
    result = initialValueArray.every(value => selectedArray.includes(value))
}
```
into
```ts
if (selectedArray.length !== initialValueArray.length)
    result = false
else 
    result = initialValueArray.every(value => selectedArray.includes(value))
```
I think maybe in this case we its so little inside the if/else we can even further rewrite this to 
`return selectedArray.length !== initialValueArray.length && initialValueArray.every(value => selectedArray.includes(value))` and skip the result variable all together. This was not obvious to me with the first version of the function. my apologies .